flusher.go

```diff
@@ -19,14 +19,17 @@ import (
 // Flush takes the slices of metrics, combines then and marshals them to json
 // for posting to Datadog.
 func (s *Server) Flush(interval time.Duration, metricLimit int) {
+	if s.IsLocal() {
+		s.FlushLocal(interval, metricLimit)
+	} else {
+		s.FlushGlobal(interval, metricLimit)
+	}
+}
+
+func (s *Server) FlushGlobal(interval time.Duration, metricLimit int) {
```

Comment:
No, this is correct. If you're a global veneur (ie you have no one to forward to) and someone sends you a local-only histogram, it's still your responsibility to flush that histogram to datadog.